src/App.tsx

```diff
@@ -0,0 +1,19 @@
+import './App.css'
+import { useQuery } from '@tanstack/react-query'
+import { ProductList } from '@/components/ProductList'
+import { getProducts } from '@/api/getProducts'
+import { Product } from '@/components/Product'
+
+function App() {
```

Comment:
We normally declare components using the arrow syntax:

```suggestion
const App = () =>  {
```